src/handler/get.ts

```diff
@@ -1,24 +1,44 @@
+import { GetParameterCommand, SSMClient } from '@aws-sdk/client-ssm';
+import type { APIGatewayProxyResult } from 'aws-lambda';
 import 'dotenv/config';
-import type { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
 import logger from '../util/logger';
 
-// eslint-disable-next-line @typescript-eslint/require-await
-export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
-  if (!event.queryStringParameters) {
-    return {
-      statusCode: 400,
-      body: JSON.stringify({
-        message: 'Missing query string parameters',
-      }),
-    };
-  }
+const headers = {
+  'Access-Control-Allow-Origin': '*',
+  'Access-Control-Allow-Headers': 'Content-Type,X-Amz-Date,Authorization,X-Api-Key,X-Amz-Security-Token',
+  'Access-Control-Allow-Methods': 'DELETE,GET,HEAD,OPTIONS,PATCH,POST,PUT',
+};
 
-  const queryParams = event.queryStringParameters;
+export const handler = async (): Promise<APIGatewayProxyResult> => {
+  try {
+    logger.info('min version endpoint called');
+    let minVersion;
 
-  logger.info(queryParams);
+    // cannot call ssm locally so this is a workaround.
+    if (process.env.AWS_SAM_LOCAL) {
+      const paramName = `${process.env.ENVIRONMENT ?? 'develop'}_vta_minimum_app_version`;
```

Comment:
Looks as if `paramName` here and on line 23 is doing the same thing. We could move this above the if check to only have one instance.

Reply:
This was just meant to be debug statements and I've gone code blind on checking it 🤦